staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5107,12 +5107,18 @@ type ReplicationControllerSpec struct {
 	// Defaults to 1.
```

Comment:
With the default value expressed declarative, we can safely remove this comment?